Projects/Domain/AuthDomain/Project.swift

```diff
@@ -8,5 +8,8 @@ let project = Project.makeModule(
     targets: [.interface, .testing, .unitTest],
     internalDependencies: [
         .Domain.BaseDomain
+    ],
+    additionalPlistRows: [
+        "BASE_URL": .string("$(BASE_URL)")
     ]
```

Comment:
```suggestion
    ]
```
이거 왜 AuthDomain에 달려있어요?